db/src/fakes/worker_manager.js

```diff
@@ -455,6 +501,10 @@ class FakeWorkerManager {
       filteredWorkerKeys.length).map(key => this.workers.get(key));
   }
 
+  async get_workers_2(worker_pool_id, worker_group, worker_id, state, page_size, page_offset) {
+    return this.get_workers(worker_pool_id, worker_group, worker_id, state, page_size, page_offset);
+  }
+
```

Comment:
Could this method be omitted?  There's no reason to need to get more than one worker secret at a time..